modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -108,7 +108,7 @@ config NRF53_MULTI_IMAGE_UPDATE
 	depends on FLASH_SIMULATOR_DOUBLE_WRITES
 	depends on !FLASH_SIMULATOR_STATS
 	depends on PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
-	depends on BOOT_UPGRADE_ONLY
+	depends on BOOT_UPGRADE_ONLY || USE_NRF53_MULIT_IMAGE_WITHOUT_UPGRADE_ONLY
```

Comment:
Did you test this ?

I'm surprised if it worked, cause there is a spelling mistake:
```suggestion
	depends on BOOT_UPGRADE_ONLY || USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY
```

Reply:
Tested it but had the spelling mistake in both my custom Kconfig ^^

Reply:
so you want to keep the spelling mistake ?

Reply:
Woops not I just haven't updated the PR :) Will update it with other changes :+1: